purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -684,6 +684,12 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
     /**
      * Automatically collect subscriber attributes associated with the device identifiers
      * $gpsAdId, $androidId, $ip
+     *
+     * @warning In accordance with [Google Play's data safety guidelines] (https://rev.cat/google-plays-data-safety),
+     * you should not be calling this function if your app targets children.
+     *
+     * @warning You must declare the [AD_ID Permission](https://rev.cat/google-advertising-id) when your app targets
```

Comment:
This link points to https://support.google.com/googleplay/android-developer/answer/6048248?hl=en